docs/_blog/announcing-anchor-outputs-channels.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing Anchor Outputs Channels"
+description: "LDK v0.0.116 has added support for opening channels with anchor outputs."
+date: "2023-08-01"
+authors:
+  - Wilmer Paulino
+tags:
+  - Channels
+  - Anchor Outputs
+  - Fees
+---
+
+The `v0.0.116` release of LDK has added support for opening channels with anchor outputs! This has
+been a much requested feature by our users as it brings channels a few security and usability
+improvements.
+
+Lightning channels rely on pre-signed transactions that its participants can broadcast to the
+network if they wish to close the channel unilaterally, e.g., when their counterparty is offline.
+Before the anchor outputs feature, participants continually negotiated their commitment
+transaction’s fees based on the current blockspace demand to ensure a timely confirmation if they
+were to be broadcast. This fee negotiation unfortunately came with its own set of problems. If
+participants disagreed on the fees proposed for a channel, the channel became unusable due to a
+unilateral close. Underestimating could result in loss of funds if an HTLC needs to be resolved
+onchain, and overestimating would allocate the majority of the channel balance towards fees,
+potentially rendering the channel useless.
+
+With anchor outputs, participants are now able to contribute the majority of fees required at the
+time of broadcast by bumping the commitment transaction’s fee via a child transaction using the
+Child-Pays-For-Parent (CPFP) fee-bumping mechanism. A small portion of fees must still be allocated
+to commitment transactions to ensure they can enter nodes’ mempools on their own. This will be
+required until package relay is deployed network-wide, at which point we can have a fixed 1 sat/vB
+commitment transaction and do away with the fee negotiation once and for all, eliminating the most
```

Comment:
I’m not so sure we’ll kill the fee negotiation mechanism as it’s coming as the downside of increased fee-bumping reserve on both sides of the channel. Though better to raised this issue on the mailing list.